src/test_encode_decode/mod.rs

```diff
@@ -191,7 +191,7 @@ fn setup_encoder<T: Pixel>(
   enc.tile_rows = 1 << tile_rows_log2;
   enc.still_picture = still_picture;
 
-  let cfg = Config::new().with_encoder_config(enc);
+  let cfg = Config::new().with_encoder_config(enc).with_threads(1);
```

Comment:
would be better to make it 2 or such for now. I'll lazy-allocate a shared pool and use it for all the tests in the future.

Reply:
Testing and fuzzing probably require different solutions since the first is single-process and the second can be multi-process. Maybe a good compromise is a simple `if cfg!(fuzzing) { 1 } else { 2 }`.